### PR TITLE
fix more typos

### PR DESCRIPTION
**Pull request by OnkarRuikar**
I found some more while working on the spell-check bot. ~Keeping it as a draft for testing and we may find some more typos.~

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Per https://github.com/mdn/content/pull/35224#issuecomment-2286442822, are you open to getting rid of as many words off the word list as possible, especially for things like variable names?

**Comment by OnkarRuikar** [comment]
> Per https://github.com/mdn/content/pull/35224#issuecomment-2286442822, are you open to getting rid of as many words off the word list as possible, especially for things like variable names?

camelCasing variable names could get tricky for code snippets referred from other example repos. Can we do this later?

File: files/en-us/web/http/status/204/index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Content-Length: 0
 ## Compatibility notes
 
 Although this status code is intended for responses with no body, servers may erroneously include data after the headers. This issue is observable in persistent connections, where the invalid body may include a distinct response to a subsequent request.
-The HTTP protocol allows browsers to handle such responses differently (there is an ongoing discussion regarding the specification text in the [HTTPWG `http-core` GitHub repository](https://github.com/httpwg/http-core/issues/26)).
+The HTTP protocol allows browsers to handle such responses differently (there is an ongoing discussion regarding the specification text in the [HTTP Working Group's `http-core` GitHub repository](https://github.com/httpwg/http-core/issues/26)).
```
Comment (Josh-Cena) [inline files/en-us/web/http/status/204/index.md:56]: We already have a lot of abbreviations in the word list. Can we add this one too?

**Review by Josh-Cena — APPROVED** [review]
Sure, one typo at a time!

Just have one question.

File: files/en-us/web/http/status/204/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/http/status/204/index.md:56]: [done](https://github.com/mdn/content/pull/35224/commits/ae7896e8361f1b59d0971457a74e740a86e6275b). This is the only text occurrence. And we ignore URLs in spell checking.

Comment (Josh-Cena) [reply files/en-us/web/http/status/204/index.md:56]:
```suggestion
The HTTP protocol allows browsers to handle such responses differently (there is an ongoing discussion regarding the specification text in the [HTTPWG `http-core` GitHub repository](https://github.com/httpwg/http-core/issues/26)).
```

**Pull request merged**